packages/examples/vite-react-typescript/src/App.css

```diff
@@ -0,0 +1,12 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;400;600;700;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bitter:wght@400;700&display=swap');
```

Comment:
I could not get the fonts to load. Ended up just doing it with Google Fonts. Is there a better way?